.mergify.yml

```diff
@@ -0,0 +1,28 @@
+pull_request_rules:
+  - name: automatic merge for Dependabot pull requests on master
+    conditions:
+      - author=dependabot[bot]
+      - status-success=continuous-integration/travis-ci/pr
+      - status-success=run_build_pipeline
+      - base=master
+    actions:
+      merge:
+        method: rebase
```

Comment:
Are we sure that any PRs can always be merged with a rebase? Do we need a [rebase_fallback](https://doc.mergify.io/actions.html#merge) setting?

Reply:
Doubtless a good idea..

Reply:
Sounds like it defaults to merge actually, so let's leave it as and take action should it break.